lambdas/aggregation/src/lambda_function.py

```diff
@@ -1,18 +1,79 @@
+import json
+
+from datetime import datetime
+
 from aws_xray_sdk.core import patch, xray_recorder
 
 from raster_analysis.exceptions import QueryParseException
 from raster_analysis.globals import LOGGER
+from raster_analysis.boto import s3_client
+
 
 patch(["boto3"])
 
 
 @xray_recorder.capture("Aggregation")
 def handler(event, context):
+    results_meta = event["distributed_map"]["ResultWriterDetails"]
     try:
+        bucket = results_meta["Bucket"]
         LOGGER.info(f"Running aggregate with parameters: {event}")
+        response = s3_client().get_object(Bucket=bucket, Key=results_meta["Key"])
+        manifest = json.loads(response["Body"].read().decode("utf-8"))
+        LOGGER.info("manifest file", manifest)
+
+        combined_data = []
+        failed_geometries = []
+        for result_record in manifest["ResultFiles"]["SUCCEEDED"]:
```

Comment:
Maybe just add a check on "FAILED", and return a fatal status if "FAILED" is not the empty list?  This seems like a very unlikely case (like an AWS system error running the processing lambda).

Reply:
So the `manifest["ResultFiles"]["FAILED"]` contains a list of the child map runs that failed and not necessarily the parent process unless the number/percent of map run failures exceeds the default or user specified threshold.  I have added the logic to parse that file and add those geometries to the failed list in case the threshold isn't exceeded. 
If the whole `ProcessList` step fails, the console shows that the aggregate step is not entered so I'm wondering if the exception blocks you put in upstream lambda function will handle that and return the failure status to the client?
This may be easier to test out when we can trigger the step function from Data API.

Reply:
Added code looks good..  Seems fine to check into `feature/otf_lists` and then once we're all integrated, we can see what kinds of errors can happen and if we're handling the common ones correctly.